rasterio/fill.py

```diff
@@ -31,7 +33,8 @@ def fillnodata(
     Parameters
     ----------
     image : numpy ndarray
-        The source containing nodata holes.
+        The source image with holes to be filled. If a MaskedArray, the
```

Comment:
@sgillies I think you should also mention `If a MaskedArray and mask is None, the inverse of its mask will...`

Otherwise, as written, it implies that it has higher precedence than the `mask` parameter.